Versioner/ObjC/AppStpreInfoObjectiveC.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  UpdateInfoObjectiveC.swift
+//  PrinceOfVersions
+//
+//  Created by Jasmin Abou Aldan on 13/09/2019.
+//  Copyright © 2019 Infinum Ltd. All rights reserved.
+//
+
+import Foundation
+
+@objcMembers
+public class AppStoreInfoObject: NSObject {
+
+
+    // MARK: - Private properties
+    private var appStoreInfo: AppStoreInfo
+
+    // MARK: - Init
+    init(from appStoreInfo: AppStoreInfo) {
+        self.appStoreInfo = appStoreInfo
+    }
+}
+
+// MARK: - Public wrappers -
+
+// Should be updated with new properties from UpdateInfo
+
+extension AppStoreInfoObject: UpdateInfoValues {
```

Comment:
Why not use swift 5.1 - keypath dynamic member lookup?

Reply:
As discussed, to use `@dynamicMemberLookup` I would need to set `appStoreInfo: AppStoreInfo` as a public property. As this is `struct`, even exposing it as `public`, Objective-C part of code will not be able to see it.

I'm leaving Swift -> ObjC wrapping as is for now